runbot_travis2docker/models/runbot_repo.py

```diff
@@ -18,3 +19,27 @@ class RunbotRepo(models.Model):
         "If is empty won't push it. "
         "Don't Use this feature if you use just one runbot server.")
     travis2docker_test_disable = fields.Boolean('Test Disable?')
+    uses_weblate = fields.Boolean()
```

Comment:
@moylop260 
I think uses_weblate is more fast to determined if the repo uses weblate.
And the view is conditioned for so weblate_url and weblate_token are required if uses_weblate is in True
The two field weblate_url and weblate_token are required for weblate api